{{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/settings.py

```diff
@@ -168,3 +169,11 @@
 
 # This is needed to get a CRSF token in /admin
 ANON_ALWAYS = True
+
+USE_X_FORWARDED_HOST = config('USE_X_FORWARDED_HOST', default=True, cast=bool)
+SECURE_SSL_REDIRECT = config('SECURE_SSL_REDIRECT', default=not DEBUG, cast=bool)
+SECURE_HSTS_SECONDS = config('SECURE_HSTS_SECONDS', default='0', cast=int)
+SECURE_HSTS_INCLUDE_SUBDOMAINS = False
```

Comment:
Why isn't this configurable? I can totally appreciate that it's False by default because otherwise you might screw up a subdomain site that is happy to be on HTTP.